src/Amplitude.php

```diff
@@ -474,49 +396,167 @@ public function getOptOut()
         return $this->optOut;
     }
 
+    /**
+     * Use the Identify API to set the User ID for a particular Device ID or update user properties
+     * of a particular user without sending an event. You can modify Amplitude default user
+     * properties as well as custom user properties that you have defined. However, these updates
+     * will only affect events going forward.
+     *
+     * Note that api key, and either the user ID or device ID need to be set prior to calling this.
+     */
+    public function identify()
+    {
+        if ($this->optOut) {
+            return $this;
+        }
+
+        $this->sendIdentify();
+
+        $this->resetUserProperties();
+    }
+
+    protected function sendIdentify()
+    {
+        $this->checkForApiKey();
+        $ch = curl_init(static::AMPLITUDE_IDENTIFY_URL);
+        $this->checkCurlHandle($ch);
+
+        $identification = [
+            'user_properties' => $this->getUserProperties()
+        ];
+
+        if ($this->getUserId()) {
+            $identification['user_id'] = $this->getUserId();
+        }
+
+        if ($this->getDeviceId()) {
+            $identification['device_id'] = $this->getDeviceId();
+        }
+
+        $postFields = [
+            'api_key' => $this->apiKey,
+            'identification' => json_encode($identification)
+        ];
+        curl_setopt($ch, \CURLOPT_POSTFIELDS, $postFields);
+        // Always return instead of outputting response!
+        curl_setopt($ch, \CURLOPT_RETURNTRANSFER, true);
+        $response = curl_exec($ch);
+        $this->checkForCurlError($ch);
+        $this->checkForHttpError($ch, $response, $postFields);
+
+        $httpCode = curl_getinfo($ch, \CURLINFO_HTTP_CODE);
+        $this->logger->log(
+            Log\LogLevel::INFO,
+            'Amplitude HTTP API response: ' . $response,
+            compact('httpCode', 'response', 'postFields')
+        );
+        curl_close($ch);
+    }
+
     /**
      * Send the event currently set in $this->event to amplitude
      *
      * Requres $this->event and $this->apiKey to be set, otherwise it throws an exception.
      *
      * @return void
-     * @throws \InternalErrorException If event or api key not set
+     * @throws AmplitudeException If event or api key not set
      */
-    protected function sendEvent()
+    protected function sendEvent(Event $event)
```

Comment:
In addition to updating the readme as I mentioned in another comment... Here, need to update the docblock.  It currently says that this method will send the event currently set in `$this->event` but that is no longer a thing in this PR...

Reply:
It would probably be good to check all the docblocks to update references throughout the file.

I know the old saying is...

> Code never lies, but comments do

But lets try to make the comments as truthful as possible  😄